types/pragma/index.d.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Parse pragma and its content from JavaScript comments
+ *
+ * @param {string} input
+ * @param {{parseContent?: (string) => any}} option
+ * @return {object}
```

Comment:
JSDoc types (eg. `{string}`) aren't required for TypeScript files:

```suggestion
```

You can also use the `@param`s to describe each parameter